packages/components/src/form-token-field/style.scss

```diff
@@ -46,10 +46,15 @@
 }
 
 .components-form-token-field__label {
-	display: inline-block;
+	display: block;
```

Comment:
This makes the label extend to all the available width, basically an area without text becomes clickable which should be avoided and the reason why `inline-block` was used. Unless I'm missing something, please revert.